frontend/app/components/css/style.js

```diff
@@ -1,28 +0,0 @@
-const style = {
-    paper: {
-        height: 400,
-        width: 400,
-        margin: 20,
-        textAlign: 'center',
-        display: 'inline-block'
-    },
-    field: {
-        margin: 10,
-        width: 'auto',
-        display: 'inline-block'
-    },
-    loginGroup: {
-        marginTop: 50,
-        marginBottom: 50
-    },
-    buttonSpacing: {
-        margin: 30
-    },
-    h1: {
-        textAlign: 'left',
-        marginLeft: '30'
-
-    }
-};
-
-module.exports = style;
```

Comment:
Add new line at the end

Reply:
this file was deleted. i applied this comment applicable files.